docs/naaccr_data.xsd

```diff
@@ -1,21 +1,26 @@
-<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:tns="http://naaccr.org/naaccrdata" targetNamespace="http://naaccr.org/naaccrdata" >
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:tns="http://naaccr.org/naaccrxml" targetNamespace="http://naaccr.org/naaccrxml" >
 
-<xsd:element name="NaaccrData">
+<xsd:simpleType name="recordtypeType">
+	<xsd:restriction base="xsd:string">
+		<xsd:enumeration value="A" />
+		<xsd:enumeration value="M" />
+		<xsd:enumeration value="C" />
+		<xsd:enumeration value="I" />
+	</xsd:restriction>
+</xsd:simpleType>
+
+<xsd:element name="NaaccrDataExchange">
 	<xsd:complexType>
 		<xsd:sequence>
-			<xsd:element ref="tns:Patient" minOccurs="0" maxOccurs="unbounded" />
+			<xsd:element ref="tns:Item" minOccurs="0" maxOccurs="unbounded" />
 			<xsd:any minOccurs="0" maxOccurs="unbounded" />
```

Comment:
<!-- DIV {margin:0px;} -->Please see discussion on basecamp
-----Original Message----- From: Fabian Depry Sent: Feb 25, 2015 3:16 PM To: depryf/naaccr-xml Cc: Roger Friedman Subject: Re: [naaccr-xml] Post-Atlanta meeting (#5) > > > - > + > Do we really want the "any" between the items and the patients? Wouldn't it be better to have it at the "end" all the time (so the "any" section for the NaaccrDataExchange tag would be after the patients, the one for the patient tag would be after the tumors and the one for the tumor tag would be after the items (since tumors doesn't have treatment yet). I don't feel strongly, but we should be consistent where it goes in those three levels, and it seems weird to have that "extra" information appear before the patients in the file...---Reply to this email directly or view it on GitHub:https://github.com/depryf/naaccr-xml/pull/5/files#r25374723